PerfNext/public/pages/index.html

```diff
@@ -6,15 +6,15 @@
       <meta name="viewport" content="width=device-width, initial-scale=1">
       <title>Perffarm Prototype</title>
```

Comment:
I just noticed, there are a couple of additional references to Perffarm Prototype that you may want to clean up also.

Reply:
Done! There is some other stuff we can clean up but I'll do that in another PR since it's not that important. My goal is to quickly add support for all third party code so that we can start using this repo directly instead of the internal repo for deployment.